pkg/driver/controller_test.go

```diff
@@ -2697,7 +2697,7 @@ func TestControllerPublishVolume(t *testing.T) {
 					if !ok {
 						t.Fatalf("Could not get error status code from error: %v", srvErr)
 					}
-					if srvErr.Code() != codes.AlreadyExists {
+					if srvErr.Code() != codes.FailedPrecondition {
```

Comment:
Can we also check if we're returning the correct instance id in the error message? It's a requirement from the spec.